config/routes.rb

```diff
@@ -13,7 +13,15 @@
     resources :rewards
   end
 
-  resources :rewards
+  resources :rewards, only: [:index]
 
   resources :users
+
+  #post "users/:id/rewards/:id", to: "redeemed_rewards#create"
+
+  #resources :users, only: [:show] do
+    #resources :redeemed_rewards, only: [:new, :create]
```

Comment:
Incorrect indentation detected (column 4 instead of 2).<br>Missing space after #.